common/src/main/java/com/wynntils/features/ui/CustomPlayerListFeature.java

```diff
@@ -0,0 +1,242 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.features.ui;
```

Comment:
I just noticed this. The feature should reside in the package corresponding to its `@Category`.